client/src/Components/TodoList.jsx

```diff
@@ -1,81 +1,84 @@
-import React, { useState, useEffect } from "react";
-import TodoForm from "./TodoForm";
-import Todo from "./Todo";
-import axios from "axios";
+import React, { useState, useEffect } from 'react';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+import axios from 'axios';
 
 function TodoList() {
-    const API_URL = process.env.REACT_APP_API_URL || "http://localhost:5000";
-    const [todos, setTodos] = useState([]);
+  const API_URL = process.env.REACT_APP_API_URL || 'http://localhost:5000';
+  const [todos, setTodos] = useState([]);
+  const token = localStorage.getItem('token');
 
   const addTodo = (todo) => {
-    if (!todo.text || /^\s*$/.test(todo.text)) {
-      return;
-    }
```

Comment:
These regex tests were actually causing your code to stop here because you are no longer using "todo.text". Instead of fixing the regex test, I decided to just remove it because it is not really helpful.